### ISSUE TITLE
Implement an ASGI app

### ISSUE THREAD

**Issue by lonelyteapot** [lonelyteapot/Transit-at-Hand-Backend#5]
An ASGI app _(Asynchronous Server Gateway Interface)_ is going to be the main server.
The GraphQL API is going to be served through it as well.

## Framework choice

Both of these are very performant and provide good developer experience.
Key feature is the easy creation of JSON APIs and OpenAPI docs, but we probably won't need it much.

###  `FastAPI`

Stable and proved by the industry.
(My opinion):
Seems to be a bit bloated.
The main maintainer is very protective and doesn't approve most pull requests.
He also doesn't support some of his other projects and they become outdated.

### `Starlite`

Basically a much better version of `FastAPI`.
It enforces good practices that make the code much better.
It's not really well-known, unfortunately.
It's very open to the community and frequently updated.
